Source/WTF/wtf/SIMDHelpers.h

```diff
@@ -116,13 +166,61 @@ ALWAYS_INLINE std::optional<uint8_t> findFirstNonZeroIndex(simde_uint16x8_t valu
         return std::nullopt;
     return std::countr_zero(mask) >> 1;
 #else
+    //  value                     |c|c|c|C|c|C|c|c| (c is character, C is matching character)
+    //  eq with charactersVector  |0|0|0|X|0|X|0|0| (X is all-1)
+    //  Reduce to uint8x8_t       |0|0|0|X|0|X|0|0| => reinterpret it as uint64_t. If it is non-zero, matching character exists.
```

Comment:
This is misleading.  Let's just say this instead:
// Incoming value is a comparison result, where each vector element is either all 1s or 0s.

Reply:
Changed.